### PR TITLE
fix: upgrading transitions

### PR DESCRIPTION
**Pull request by renlulu**

### CODE REVIEW

**Review by bb111189 — APPROVED** [review]

File: contracts/README.md [hunk 1]
```diff
@@ -284,9 +284,13 @@ Each of these category of transitions are presented in further detail below.
...
 | `UpdateDeleg` | `ssnaddr: ByStr20, deleg : ByStr20, stake_amt: Uint128, initiator: ByStr20`| To add or remove a delegator for an SSN. <br>  :warning: **Note:** `initiator` must be the current `contractadmin` of the contract.  | :heavy_check_mark: | :heavy_check_mark: |
-| `PopulateStakeSSNPerCycle` | `ssn_addr: ByStr20, cycle: Uint32, info: SSNCycleInfo, initiator: ByStr20`| To populate `stake_ssn_per_cycle` map. <br>  :warning: **Note:** `initiator` must be the current `contractadmin` of the contract.  | :heavy_check_mark: | <center>:x:</center> |
+| `PopulateStakeSSNPerCycle` | `ssn_addr: ByStr20, cycle: Uint32, totalAmt: Uint128, rewards: Uint128, initiator: ByStr20`| To populate `stake_ssn_per_cycle` map. <br>  :warning: **Note:** `initiator` must be the current `contractadmin` of the contract.  | :heavy_check_mark: | <center>:x:</center> |
```
Comment (AmritKumar) [inline contracts/README.md:287]: Not sure why this was necessary. Is it just simplify message.json?

File: contracts/README.md [hunk 2]
```diff
@@ -531,8 +539,9 @@ The first transition is meant to submit request for transfer of native ZILs whil
...
-|`SubmitPopulateStakeSSNPerCycleTransaction`| `calleeContract : ByStr20, ssn_address : ByStr20, cycle : Uint32, info : SSNCycleInfo` | Submit a request to invoke the `PopulateStakeSSNPerCycle` transition in the `SSNListProxy` contract. |
+|`SubmitPopulateStakeSSNPerCycleTransaction`| `calleeContract : ByStr20, ssn_address : ByStr20, cycle : Uint32, totalAmt: Uint128, rewards: Uint128` | Submit a request to invoke the `PopulateStakeSSNPerCycle` transition in the `SSNListProxy` contract. |
 |`SubmitPopulateLastWithdrawCycleFordelegTransaction`| `calleeContract : ByStr20, deleg_address : ByStr20, ssn_address : ByStr20, cycle : Uint32` | Submit a request to invoke the `PopulateLastWithdrawCycleFordeleg` transition in the `SSNListProxy` contract. |
+|`SubmitPopulateLastBufDepositCycleDelegTransaction`| `calleeContract : ByStr20, deleg_address : ByStr20, ssn_address : ByStr20, cycle : Uint32` | Submit a request to invoke the `PopulateLastBufDepositCycleDeleg` transition in the `SSNListProxy` contract. |
```
Comment (AmritKumar) [inline contracts/README.md:544]: Why multisig README has only 1 new transition when a total of 4 new were added in the SSNList contract?

**Review by AmritKumar — CHANGES_REQUESTED** [review]

File: contracts/README.md (hunk 1 above)
Comment (renlulu) [reply contracts/README.md:287]: Yes

File: contracts/README.md (hunk 2 above)
Comment (renlulu) [reply contracts/README.md:544]: Opps added other 3

**Review by bb111189 — APPROVED** [review]

**Review by AmritKumar — APPROVED** [review]

**Pull request merged**